libraries/Microsoft.Bot.Builder/TurnContextServiceCollection.cs

```diff
@@ -46,95 +22,44 @@ public TurnContextServiceCollection()
         {
             if (key == null) throw new ArgumentNullException(nameof(key));
```

Comment:
consider changing to follow our coding convension, and put the throw on it's own line.